src/import/prepare.rs

```diff
@@ -72,16 +76,9 @@ impl<Client: ImportClient> PrepareJob<Client> {
             }
         };
 
+        IMPORT_EACH_PHASE.with_label_values(&["prepare"]).set(1.0);
```

Comment:
You mean `inc()` ? How many jobs will run concurrently?

Reply:
1. No, it's just a flag to indicate the current import phase.
2. There will not run concurrently, only one import job at anytime.